modules/indexer_monitors/monitors.tf

```diff
@@ -0,0 +1,238 @@
+resource "datadog_monitor_json" "best_bid" {
+  monitor = <<EOF
+{
+	"name": "[${var.environment}] Indexer Orderbook best bid has not changed for 10 minutes for pair {{clob_pair_id.name}}",
+	"type": "query alert",
+	"query": "max(last_10m):abs(derivative(avg:roundtable.crossed_orderbook.best_bid_human{env:${var.environment},!clob_pair_id:33} by {clob_pair_id})) <= 0",
+  "message": "
+
+{{#is_alert}}
+${var.pagerduty_tag}
+{{/is_alert}} 
+
+{{#is_recovery}}
+${var.pagerduty_tag}
+{{/is_recovery}} 
+
+${var.slack_channel}
+",
+	"tags": [
+		"team:${var.team}",
+		"env:${var.env_tag}"
+	],
+	"options": {
+		"thresholds": {
+			"critical": 0
+		},
+		"notify_audit": false,
+		"require_full_window": false,
+		"notify_no_data": false,
+		"renotify_interval": 0,
+		"include_tags": true,
+		"new_group_delay": 60
+	},
+	"priority": null,
+	"restricted_roles": null
+}
+EOF
+}
+
+resource "datadog_monitor_json" "best_ask" {
+  monitor = <<EOF
+{
+	"id": 127134448,
+	"name": "[${var.environment}] Indexer Orderbook best ask has not changed for 10 minutes for pair {{clob_pair_id.name}}",
+	"type": "query alert",
+	"query": "max(last_10m):abs(derivative(avg:roundtable.crossed_orderbook.best_ask_human{env:${var.environment},!clob_pair_id:33} by {clob_pair_id})) <= 0",
+  "message": "
+{{#is_alert}}
+${var.pagerduty_tag}
+{{/is_alert}} 
+
+{{#is_recovery}}
+${var.pagerduty_tag}
+{{/is_recovery}} 
+
+${var.slack_channel}
+",
+	"tags": [
+		"team:${var.team}",
+		"env:${var.env_tag}"
+	],
+	"options": {
+		"thresholds": {
+			"critical": 0
+		},
+		"notify_audit": false,
+		"require_full_window": false,
+		"notify_no_data": false,
+		"renotify_interval": 0,
+		"include_tags": true,
+		"new_group_delay": 60
+	},
+	"priority": null,
+	"restricted_roles": null
+}
+EOF
+}
+
+resource "datadog_monitor_json" "socks_kafka_offset" {
+  monitor = <<EOF
+{
+	"id": 117804982,
+	"name": "[${var.environment}] Indexer Socks has high kafka offset",
+	"type": "query alert",
+	"query": "min(last_10m):top(avg:aws.kafka.max_offset_lag{cluster_name:${var.msk_cluster_name},consumer_group:socks_*} by {consumer_group}.fill(last), 5, 'mean', 'asc') > 1000",
+	"message": "Max Kafka Offset is > 1000 for at least 1 socks instance. This means delayed notifications for all websocket messages.\n\n@slack-alerts-indexer-staging",
```

Comment:
This should have the slack channel variable, and shouldn't be the escaped string.